rmf_schedule_visualizer/src/main.cpp

```diff
@@ -61,9 +61,22 @@ int main(int argc, char* argv[])
   const uint16_t port = port_string.empty() ? 8006 : std::stoul(
     port_string, nullptr, 0);
 
+  std::string retained_history_count_str;
+  uint retained_history_count = 0;
+  if (get_arg(args, "-history", retained_history_count_str,
```

Comment:
I won't pretend to be a CLI expert, but my impression is that `-flag` is kind of a dated format for CLI flags.

I think in modern CLI conventions, whole-word arguments are usually given double-dashes while single-dashes are reserved for single-letter flag shortcuts. I.e. we would have `--history` or `-h` as equivalent flags.

Reply:
`-h` can be mistaken for help, should we just set it to `--history`?

Reply:
Great point! That sounds good to me.

Reply:
e13db07